File: src/tests/Markify.Fixtures/Projects/Source/Interface/InterfaceMethods.cs
public interface FooType
{
    void Method();

    int IntMethod();

    void WithParametersMethod(int foo, ref int bar, out int foobar);

    void WithNoNameParameterMethod(__arglist)

    T SingleGenericMethod<T>(T foo = default(T));

    T MultiGenericMethod<T, Y>(T foo, Y bar)
        where T : IList
        where Y : IDisposable, IEnumerable;
}

File: src/Services/Roslyn/Markify.Services.Roslyn.Fixtures/Projects/Source/Interface/InterfaceMethods.vb
Public Interface FooType
    Sub Method()

    Function IntMethod() As Integer

    Sub WithParametersMethod(foo As Integer, ByRef bar As Integer, ByVal foobar As Integer)

    Function SingleGenericMethod(Of T)(Optional foo As T = Nothing) As T

    Function MultiGenericMethod(Of T As IList, Y As {IDisposable, IEnumerable})(foo As T, bar As y) As T
End Interface